### ISSUE TITLE
Support for Multiple Mowers

### ISSUE THREAD

**Issue by normen** [weweave/landroid-bridge#25]
How are multiple mowers in the same account handled? Are the commands issued to all of the mowers at once? Is there anything I can do to help support multiple mowers better?

The way it looks now I think having multiple servers each with a "dev_sel": X entry in the landroid-s section of the config.json is the way to go?

**Comment by virtualzone** [comment]
Hi @normen, I've added a hint to the Readme on how to manage multiple mowers. You'll have to run one instance of the bridge per mower, setting the right dev_sel property per instance. Furthermore, you'll need to set HTTP Port and MQTT Topic to unique values. Hope this helps.

**Comment by normen** [comment]
Yeah, thats what I did. Don't find that especially elegant though. Simply adding multiple mowers to one instance of the server with different paths sounds much better to me.

But thanks for this bridge nonetheless, great work! If I actually get around to adding support for multiple mowers would you be interested in a PR? Or is this going completely out of the scope for the plugin.

My idea was to simply have multiple "landroid-s" sections, each with a unique name and the app would simply use all first level json objects that DO have the needed data as new landroids. That way it'd be backwards compatible when using the "landroid-s" name and for multiple mowers you could simply call them any way you like.

**Comment by normen** [comment]
I didn't look too much into the code so I can't say how complicated it would actually be, it looked easy enough on the first overview. But since I have a solution for myself now (two landroid bridges and a HTTP plugin for HomeBridge) I guess my drive to get this working won't be too intense.

I did however think about wrapping this server into a HomeBridge plugin to get more functionality exposed to HomeKit like battery status and an actual display of what it's doing. HomeBridge is a HomeKit bridging server to integrate basically any device into HomeKit. If I actually get around to doing that I'll also look into adding multi-mower support.

As for how many people would need this I really can't say. To me it seems obvious you might have more than one Landroid but I am living in a rural area and have lots of lawn :)
